.github/workflows/release.yml

```diff
@@ -85,10 +93,10 @@ jobs:
 
           # we usually do not update nightly files during major releases
           if [[ ${CHE_VERSION} == *".0" ]]; then
-            ./make-release.sh ${CHE_VERSION} --release --release-olm-files --update-nightly-olm-files 
+            ./make-release.sh ${CHE_VERSION} --release --release-olm-files --update-nightly-olm-files --dev-workspace-controller-version ${DWO_VERSION} --dev-workspace-che-operator-version ${DWO_CHE_VERSION}
           else
             git checkout ${BRANCH}
-            bash -x ./make-release.sh ${CHE_VERSION} --release --release-olm-files
+            bash -x ./make-release.sh ${CHE_VERSION} --release --release-olm-files --dev-workspace-controller-version ${DWO_VERSION} --dev-workspace-che-operator-version ${DWO_CHE_VERSION}
```

Comment:
why are we `bash -x` here when two lines above we're not? some reason you want more debugging for this step?